examples/proxy-server/src/main/java/example/armeria/proxy/ProxyService.java

```diff
@@ -0,0 +1,124 @@
+import com.linecorp.armeria.common.HttpRequest;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.SessionProtocol;
+import com.linecorp.armeria.server.AbstractHttpService;
+import com.linecorp.armeria.server.ServiceRequestContext;
+
+public final class ProxyService extends AbstractHttpService {
+
+    /**
+     * We used hardcoded backend addresses. But you can use other service discovery mechanisms to configure
+     * backends dynamically using {@link DnsServiceEndpointGroup}, ZooKeeper or Central Dogma.
+     * See <a href="https://line.github.io/armeria/client-service-discovery.html">service discovery</a>,
+     * <a href="https://line.github.io/armeria/advanced-zookeeper.html#advanced-zookeeper">Service discovery
+     * with ZooKeeper</a> and <a href="https://line.github.io/centraldogma/">centraldogma</a>.
+     */
+    private static final EndpointGroup animationGroup = new StaticEndpointGroup(
+            Endpoint.of("127.0.0.1", 8081),
+            Endpoint.of("127.0.0.1", 8082),
+            Endpoint.of("127.0.0.1", 8083));
+
+    private final HttpClient loadBalancingClient;
+
+    private final boolean filterHeaders;
+
+    public ProxyService() throws InterruptedException {
+        loadBalancingClient = newLoadBalancingClient();
+        filterHeaders = false;
+    }
+
+    private static HttpClient newLoadBalancingClient() throws InterruptedException {
+        // Send HTTP health check requests to '/internal/l7check' every 10 seconds.
+        final HttpHealthCheckedEndpointGroup healthCheckedGroup =
+                new HttpHealthCheckedEndpointGroupBuilder(animationGroup, "/internal/l7check")
+                        .protocol(SessionProtocol.HTTP)
+                        .retryInterval(Duration.ofSeconds(10))
+                        .build();
+
+        // Wait until the initial health check is finished.
+        healthCheckedGroup.awaitInitialEndpoints();
+
+        EndpointGroupRegistry.register("animation_apis", healthCheckedGroup,
+                                       // You can use EndpointSelectionStrategy.WEIGHTED_ROUND_ROBIN or even
+                                       // implement your own strategy to balance requests.
+                                       EndpointSelectionStrategy.ROUND_ROBIN);
+
+        return new HttpClientBuilder("http://group:animation_apis")
+                // Disable timeout to serve infinite streaming response.
+                .defaultResponseTimeoutMillis(0)
+                .build();
+    }
+
+    @Override
+    protected HttpResponse doGet(ServiceRequestContext ctx, HttpRequest req) throws Exception {
+        // We can just send the request from a browser to a backend and the response from the backend
+        // to the browser. You don't have to implement your own backpressure control because Armeria handles
+        // it by nature. Let's take a look at how a response is sending to describe it.
```

Comment:
How about just:
`Let's take a look at how a response is sending to describe it. This is the flow:`
->   
`Let's take a look at how a response is transferring from the backend to the browser:`?

Reply:
All fixed. PTAL. 🙇